<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>拷贝图片文件</string>
  </property>
  <widget class="QTextEdit" name="enginePath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selPathBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>540</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>sel path</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>目标路径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>前缀：</string>
   </property>
  </widget>
  <widget class="QLabel" name="img1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图片1</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="img_path1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="placeholderText">
    <string>图片1</string>
   </property>
  </widget>
  <widget class="QLabel" name="img2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图片2</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="img_path2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>图片2</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="img_path3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>图片3</string>
   </property>
  </widget>
  <widget class="QLabel" name="img3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图片3</string>
   </property>
  </widget>
  <widget class="QLabel" name="img4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图片4</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="img_path4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="placeholderText">
    <string>图片4</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="img_path5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>图片5</string>
   </property>
  </widget>
  <widget class="QLabel" name="img6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图片6</string>
   </property>
  </widget>
  <widget class="QLabel" name="img5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图片5</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="img_path6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>图片6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_all_img">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空所有图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="do_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>执行拷贝</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="preNameEd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_some_img">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空1,2,3,6图</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
